index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <p>Hello World, we are software developers!</p>
```

Comment:
I suggest you use more appropriate tags in your HTML. Since this is the heading of your webpage, it would be better if you use any tags from `<h1>` to `<h6>` and also enclose it under the `<header>` tag.